src/lib/utils/hooks/useGetEntityHeaders.ts

```diff
@@ -30,6 +41,17 @@ export default function useGetEntityHeaders(props: UseGetProfilesProps) {
           const newReferencesChunks = chunk(newReferences, 45)
           for (const newReferences of newReferencesChunks) {
             const newData = await getAllEntityHeader(newReferences, token)
+            const notFound = newReferences.filter(
+              item => newData.map(item => item.id).indexOf(item.targetId) == -1,
+            )
+            const notFoundPlaceholders: EntityHeader[] = notFound.map(item => ({
+              ...entityHeaderTemplate,
+              id: item.targetId,
+              name: `Unknown/Unable to Access (${item.targetId})`
+            }))
+            setData(oldData =>
+              oldData.concat(...newData, ...notFoundPlaceholders),
+            )
             setData(oldData => oldData.concat(...newData))
```

Comment:
this line can get removed with the addition of the line above

Reply:
Good catch!
Thank you. Had it stashed.. and didn't quite resolve the conflicts correctly. Fixed